src/client/component/command.cpp

```diff
@@ -239,7 +239,23 @@ namespace command
 				}
 				else
 				{
-					const auto wp = game::G_GetWeaponForName(arg.data());
+					std::string weaponName = arg.data();
```

Comment:
also, std::string definition when you're doing .data()? isn't arg already a std::string